lib/app/endpoint/home.rb

```diff
@@ -164,46 +171,45 @@ class Home < Endpoint
           end
         end
 
-        # Handles EHR Launch and any redirects
         get '/:id/:key/:endpoint/?' do
           instance = Inferno::Models::TestingInstance.get(params[:id])
-          halt 404 unless !instance.nil? && instance.client_endpoint_key == params[:key] && %w[launch redirect].include?(params[:endpoint])
+          halt 404 unless !instance.nil? && instance.client_endpoint_key == params[:key] && ['launch','redirect'].include?(params[:endpoint])
```

Comment:
The [default RuboCop configuration](https://rubocop.readthedocs.io/en/latest/cops_style/#stylewordarray) prefers [Percent Strings](https://docs.ruby-lang.org/en/2.5.0/syntax/literals_rdoc.html#label-Percent+Strings), which can be beneficial if we want to ensure that the array only contains String literals:
```suggestion
          halt 404 unless !instance.nil? && instance.client_endpoint_key == params[:key] && %w[launch redirect].include?(params[:endpoint])
```
If we want to go with the other syntax then we should change the RuboCop configuration to use `EnforcedStyle: brackets` for consistency across the application